ddtrace/internal/datadog/profiling/dd_wrapper/include/types.hpp

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+namespace Datadog {
+enum SampleType : unsigned int
+{
+    CPU = 1 << 0,
+    Wall = 1 << 1,
+    Exception = 1 << 2,
+    LockAcquire = 1 << 3,
+    LockRelease = 1 << 4,
+    Allocation = 1 << 5,
+    Heap = 1 << 6,
+    All = CPU | Wall | Exception | LockAcquire | LockRelease | Allocation | Heap
+};
+
+// Rather than wrapping and unwrapping values when they're flushed, we just use a vector.
```

Comment:
This is a struct not a vector

Reply:
Indeed--it's a struct for indices into the vector.  Updated the comment.